data_base.py

```diff
@@ -0,0 +1,67 @@
+import sqlite3
+from functools import wraps as wraps
+import random
+
+
+def work_with_db(file):
+    def dec(func):
+        @wraps(func)
+        def decorator(*args, **kwargs):
+            with sqlite3.connect(file) as dbd:
+                dbd.cursor()
+                dbd.execute('''CREATE TABLE IF NOT EXISTS photos_list 
+                                (id INTEGER, 
+                                url VARCHAR,
+                                name VARCHAR,
+                                posted BOOL, 
+                                UNIQUE(id) )''')
+                return func(dbd, *args, **kwargs)
+
+        return decorator
+
+    return dec
+
+
+@work_with_db("test.db")
+def add_photo(db, attributes):
```

Comment:
This method is closely coupled with "attributes" structure. I think it would be better to pass each fragment as a separate argument.